Google.PowerShell/BigQuery/BqTable.cs

```diff
@@ -163,6 +163,63 @@ public void DoGetRequest(TablesResource.GetRequest request)
 
     /// <summary>
     /// <para type="synopsis">
+    /// Updates information describing an existing BigQuery table.
+    /// </para>
+    /// <para type="description">
+    /// text
```

Comment:
Agreed.  The description is not so useful when the coder forgets to copy-paste it out of the design doc.